src/muya/lib/utils/exportMarkdown.js

```diff
@@ -256,11 +256,14 @@ class ExportMarkdown {
     const { row, column } = table
     const tableData = []
     const tHeader = table.children[0]
-
     const tBody = table.children[1]
-    tableData.push(tHeader.children[0].children.map(th => th.text.trim()))
+    const escapeText = str => {
+      return str.replace(/\|/g, '\\|')
```

Comment:
I think you should match all `|` that are not preceded by the escape character `\` in a table cell.

Reply:
@fxha Fixed, pls review again.